geometry_msgs/Point target_position
float32 effector_angle
int32 solution
